### PR TITLE
Chore/suite absolute imports

### PR DESCRIPTION
**Pull request by mroz22**
Please lets get rid of import aliases as they do more harm than good. IMHO readability does not benefit from aliases at all. One has to think whether import starting with '@' is actually referencing another package in monorepo, or some subfolder in suite. Also time to time there are other problems with them. I was about to update jest form packages/suite but kept getting some weird errors related to aliases (jest-utils resolved somewhere in suite). We typically figure out and fix but I would argue that this does not really pay off in the long run.

5f8f620 - all configuration changes
1667455 - only imports changes resulting from configuration changes. should be pure refactoring
fba452d - changed one default export to named export. I still dont understand why but it seems that it fixed this CI error https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/4494717063

### CODE REVIEW

File: tsconfig.aliases.json [hunk 1]
```diff
@@ -1,135 +1,4 @@
 {
```
Comment (Nodonisko) [inline tsconfig.aliases.json]: You can remove this file completely and use just `tsconfig.json` from repo root.

**Review by Nodonisko — APPROVED** [review]

File: tsconfig.aliases.json (hunk 1 above)
Comment (mroz22) [reply tsconfig.aliases.json]: [0437324](https://github.com/trezor/trezor-suite/pull/8723/commits/043732481917d35390cb67116a06f678942e7818)

**Review by matejkriz — APPROVED** [review]

**Review by tomasklim — APPROVED** [review]
For me, imports work the same as before

**Pull request merged**